README.md

```diff
@@ -22,7 +22,7 @@ and [OpenAPI 3.1.x](https://github.com/OAI/OpenAPI-Specification/blob/master/ver
 
   You should regenerate the TOC after making changes to this file.
 
-      npm run format-readme-toc
+      npx markdown-toc --maxdepth 5 -i README.md
```

Comment:
Since you updated the script, this file doesn't actually need to change - right?

Reply:
correct, fixed it.